actions/run_action.go

```diff
@@ -113,7 +113,13 @@ func (run *RunAction) doRun(context debos.DebosContext) error {
 		label = path.Base(run.Script)
 	} else {
 		cmdline = []string{run.Command}
-		label = run.Command
+		commands := strings.Split(run.Command, "\n")
+		label = commands[0]
+
+		// Make it clear a multi-line command is being run
+		if len(commands) > 1 {
+			label += "..."
```

Comment:
perhaps we should trim the label to something like 40 characters before applying `...` ?

Reply:
Is there an existing idiom for this elsewhere in the code I can reuse?

Reply:
how about something like (untested)

```
commands := strings.Split(run.Command, "\n")

// Make it clear a multi-line command is being run
let maxLabelLength := 40
if len(commands[0]) > maxLabelLength || len(commands) > 1 {
	label = commands[0][:maxLabelLength]
	label = strings.Trim(label)
	label += "..."
}
```

Reply:
As I understand, slicing a string past its end is an error in Go.

Reply:
also, `let` is a rustism 🙂

Reply:
Your suggestion seems okay, but maybe it'd be better to truncate long labels elsewhere, in the log printer?